doc/source/development/maintaining.rst

```diff
@@ -368,11 +368,11 @@ Prerequisites
 
 In order to be able to release a new pandas version, the next permissions are needed:
 
-- Merge rights to the [pandas](https://github.com/pandas-dev/pandas/),
-  [pandas-wheels](https://github.com/MacPython/pandas-wheels), and
-  [pandas-feedstock](https://github.com/conda-forge/pandas-feedstock/) repositories.
+- Merge rights to the `pandas <https://github.com/pandas-dev/pandas/>`_,
+  `pandas-wheels <https://github.com/MacPython/pandas-wheels>`_, and
+  `pandas-feedstock <https://github.com/conda-forge/pandas-feedstock/>`_ repositories.
 - Permissions to push to main in the pandas repository, to push the new tags.
-- Write permissions to [PyPI](https://github.com/conda-forge/pandas-feedstock/pulls)
+- Write permissions to `PyPI <https://github.com/conda-forge/pandas-feedstock/pulls>`_
```

Comment:
```suggestion
- `Write permissions to PyPI <https://github.com/conda-forge/pandas-feedstock/pulls>`_
```

To avoid

```
2023-01-20T05:53:13.6374953Z /home/runner/work/pandas/pandas/doc/source/development/maintaining.rst:5: WARNING: Duplicate explicit target name: "pypi".
```

Reply:
Thanks @mroeschke! I will make the change rn